centaur/src/main/resources/standardTestCases/cwl_secondary_files/cwl_secondary_files.cwl

```diff
@@ -0,0 +1,30 @@
+cwlVersion: v1.0
+$graph:
+- id: main
+  class: CommandLineTool
+  hints:
+    DockerRequirement:
+      dockerPull: "debian:stretch-slim"
+  inputs:
+    - id: command
+      type: string
+    - id: f
+      type: File
+      inputBinding:
+        position: 2
+      secondaryFiles: [.also]
+    - id: fs
+      type:
+        type: array
+        items: File
+        inputBinding:
+          position: 3
+      secondaryFiles: [.also]
+  outputs:
+    ignored:
+      type: int
+      outputBinding:
+        glob: "*.txt"
+        outputEval: $(self.length)
```

Comment:
this is a goofy bit of vestigial code and could maybe just return a constant integer expression or something

Reply:
Note: I wrote this bit and am the source of the goofiness.